src/sensors/RealSenseCamera.cc

```diff
@@ -41,6 +54,10 @@ RealSenseCamera::RealSenseCamera(size_t width, size_t height, unsigned int fps)
 
     this->scale = this->dev->get_depth_scale();
 
+    if (this->dev->supports_option(r200_opts[0])) {
+        this->dev->set_options(r200_opts, ARRAY_SIZE(r200_opts), r200_values);
```

Comment:
Why are you defining the options and values arrays as static? Isn't better to define them as local variables? I took a look at the set_options code and it looks like it is copying the values of the elements of the arrays, so we don't need to keep them as static variables.
Have you checked that?